src/Exception/EvaluatorIsNotAllowedException.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Overblog\GraphQLBundle\Exception;
```

Comment:
this should be moved in ExpressionLanguage namespace

Reply:
Will do it.
I guess the `ArgumentsValidationException` class should be moved to Validator namespace too?

Reply:
yes an Exception dir should be create under each related Namespace if possible